ts/components/conversation/ConversationHeader.tsx

```diff
@@ -703,7 +703,10 @@ export class ConversationHeader extends React.Component<PropsType, StateType> {
       default:
         throw missingCaseError(type);
     }
-
+    return onClick;
+  }
+  private renderHeader(): ReactNode {
```

Comment:
Same as above, I think we can revert this change and keep `onClick` generation as part of `renderHeader()`. Then we can pass the `onClick` to `renderAvatar()`